share/dkg/dkg.go

```diff
@@ -793,6 +806,11 @@ func (d *DistKeyGenerator) ProcessJustifications(bundles []*JustificationBundle)
 		}
 	}
 
+	// check if we are evicted or not
+	if err := d.checkIfEvicted(JustifPhase); err != nil {
```

Comment:
I think this shadows the named return `err` when using `:=` - albeit it's the only time `err` is set, so perhaps the named return can be removed

Reply:
It totally can and has been removed !